docker-connector-demo/processes/dockerDemo.p.json

```diff
@@ -41,7 +41,7 @@
           "code" : [
             "import com.axonivy.connector.docker.PortBinding;",
             "PortBinding binding = new PortBinding();",
-            "binding.publicPort = 8080;",
+            "binding.publicPort = 9090;",

```

Comment:
wouldn't that be a good candidate for 'ivy.var' ? so that the user can configure the preferred http port to be bound?